frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -232,14 +254,42 @@ private static JBColor getColor(SyncToRemoteStatus relation) {
 
   private static String getLabel(SyncToRemoteStatus status) {
     var remoteName = status.getRemoteName();
-    return Match(status.getRelation()).of(
+    return "  " + Match(status.getRelation()).of(
```

Comment:
Nit: for other labels, it looks like it's rather the **c'tor** that's responsible for adding the proper whitespace

Reply:
Yes, you're right